index.html

```diff
@@ -9,9 +9,29 @@
 
     <link rel="stylesheet" href="styles.css" />
 
+    <script src="utlities.js" defer></script>
     <script src="app.js" defer></script>
+    
   </head>
   <body>
-      <h1>Surprise</h1>
+    <h1>Surprise</h1>
+
+    <form id="choose-ingredient-form">
```

Comment:
I would get rid of the form with a submit button and go ahead with a `change` event, something like in the first answer [here](https://stackoverflow.com/questions/8838648/onchange-event-handler-for-radio-button-input-type-radio-doesnt-work-as-one).

Reply:
ok